app/models/user.rb

```diff
@@ -105,6 +105,21 @@ def role_psm? project
                         user_id: id)
   end
 
+  def projects_role_psm
+    role = Role.find_by(name: Settings.project_roles.PSM)
+
+    if role
+      projects = Project.joins(:project_users).where(project_users: {
+                                                       user_id: id,
+                                                       project_role_id: role.id
+                                                     })
+    else
+      projects = []
+    end
+
+    projects
```

Comment:
```suggestion
    projects = []
    if role
      projects = Project.joins(:project_users).where(project_users: {
                                                       user_id: id,
                                                       project_role_id: role.id
                                                     })
    end

    projects
```
ri thì ko cần else 

nhưng chị có thắc mắc dữ lắm

sao cần find_by rồi handle thế này nhỉ, chưa hiểu mục đích và ko thấy chỗ sử dụng luôn

Reply:
hàm này bị trùng với bạn khác đã làm mà em quên chưa bỏ đi